0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -199,12 +208,19 @@ Submit a fulfillment to a ledger. Plugin must be connected, otherwise the promis
 
 **TODO**: Define what the message format is.  Plugin must be connected, otherwise the promise should reject.
 
+#### rejectIncomingTransfer
+<code>ledgerPlugin.rejectIncomingTransfer( **transferId**:String ) ⇒ Promise.&lt;null></code>
+
+Reject an incoming transfer that is held pending the fulfillment of its `executionCondition` before the `expiresAt` time.
+
```

Comment:
Does this mean the protocol doesn't allow ledgers to reject a transfer that is unconditional?